I finished my minigame work, including a new minigame that prints out

goliaro opened the pull request:
the latest tweets of a specific account on Twitter.

goliaro committed:
I finished my minigame work, including a new minigame that prints out
the latest tweets of a specific account on Twitter.

ajamesVISD commented:
Cannot change the Minigame interface, other students' code relies on it. If you're going to handle exceptions you have to do that within your code.

goliaro committed:
I restored the interface to its original version without any exception
throw. I handled the exceptions (IOException and TwitterException) into
the Twitter Minigame class

goliaro committed:
removed the remaining exception throws

goliaro committed:
.
